cmd/image/qcow2ova/prep/templates.go

```diff
@@ -66,7 +66,9 @@ do
 done
 grub2-mkconfig -o /boot/grub2/grub.cfg
 rm -rf /etc/sysconfig/network-scripts/ifcfg-eth0
+{{if .RootPasswd }}
```

Comment:
This will never be empty per logic we have here - https://github.com/ppc64le-cloud/pvsadm/blob/1c5b59e51d54a7795b6f6146c6f007858a115c4d/cmd/image/qcow2ova/qcow2ova.go#L127:L134

Need to find a way to keep this empty when user doesn't want to set a password

Reply:
Sure, will add a flag "skip-os-password" for user to specify password is not required.

Reply:
make sense, lets add that flag.